supabase-ws-flutter-mobile/lib/core/source/auth_remote_source.dart

```diff
@@ -1,13 +1,15 @@
 import 'package:flutter_template/core/model/service/auth_models.dart';
 import 'package:flutter_template/core/model/service/service_response.dart';
 import 'package:flutter_template/core/source/common/http_service.dart';
+import 'package:supabase_flutter/supabase_flutter.dart';
 
 class AuthRemoteSource {
```

Comment:
let split the AuthRemoteSource in two, define an interface and the implementation. It will be easier to explain